README.md

```diff
@@ -2,7 +2,77 @@
 
 FastAPI-based REST API for sending emails
 
-----
+## Run it
+
+```bash
+docker compose up
+```
+
+## Development
+
+### Install
+
+```bash
+pip install poetry
+poetry install
+poetry shell
+```
+
+### Testing
+
+```bash
+pytest
+```
+
+## API
+
+### `GET /`
+
+```bash
+curl http://localhost:8765
+```
+```json
+{"message":"Welcome to the Email API"}
+```
+
+### `POST /send`
+
+```bash
+curl -vs -X POST \
+-H "X-Mail-Subject: Institution Profile Change" \
+-H "X-Mail-Sender-Address: jane.doe@some.org" \
+-H "X-Mail-Sender-Name: Jane Doe" \
```

Comment:
I'm guessing using headers here is to make it more general purpose? We'd need to make note and communicate with front end that it's not just a straight form post of all the fields on the form; they will need to use these form fields as headers, and the rest of the form fields as the normal form post.

Reply:
Yep. I chatted with the frontend folks today. I think we're all good for an initial version. @shindigira spun up issue https://github.com/cfpb/sbl-frontend/issues/244 to implement it on the frontend.

One of the next steps, though, is to add OIDC auth to the service. Once that's done, the two `Sender` headers can be replaced on the backend with content from the OIDC access token.

The `X-Mail-Subject` header is probably here to stay...at least for the immediate future. For now, the frontend is going to set that value to the name of the form that's submitting it.